Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2_test.py

```diff
@@ -468,16 +468,16 @@
         'Address': '98.136.103.23', 'ASN': 'AS36647 YAHOO-GQ1', 'Geo': {'Location': '37.751:-97.822', 'Country': 'US'},
         'Relationships': [{'Relationship': 'indicator-of', 'EntityA': '98.136.103.23', 'EntityAType': 'IP',
                            'EntityB': 'T1140 - Deobfuscate/Decode Files or Information',
-                           'EntityBType': 'STIX Attack Pattern'},
+                           'EntityBType': 'Attack Pattern'},
```

Comment:
These changes are not related to you.
The initiative is welcome, but please confirm these changes are correct.